spotirec.py

```diff
@@ -23,8 +24,12 @@
 
 sp_oauth = oauth2.SpotifyOAuth(client_id, client_secret, redirect_uri, scopes=scope, cache=cache)
 
-parser = argparse.ArgumentParser(epilog='passing no optional arguments defaults to basing recommendations off the '
-                                        'user\'s top genres')
+parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter,
+                                 epilog=dedent("""
+                                 passing no optional arguments defaults to basing recommendations
+                                 off the user\'s top genres\nspotirec is released under GPL-3.0 and comes with 
```

Comment:
The newline seems to botch output formatting a bit.
```
                                 passing no optional arguments defaults to basing recommendations
                                 off the user's top genres
spotirec is released under GPL-3.0 and comes with 
                                 ABSOLUTELY NO WARRANTY, for details read LICENSE
```

Reply:
Yep, fixed in 5515973. I actually wrote the issue in the commit msg :)